.pylintrc

```diff
@@ -140,63 +163,33 @@ property-classes=abc.abstractproperty
 # Regular expression matching correct module names
 module-rgx=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
 
-# Naming hint for module names
-module-name-hint=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
```

Comment:
These `foo-name-hint` options were superseded by `foo-rgx`, which we already have.